src/main/resources/worker.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Downloads and runs an analyst worker.
+# This will be run through Java MessageFormat, and the following items will be available by wrapping
+# the relevant index in curly braces
```

Comment:
I'm not sure someone reading this comment will understand "Available by wrapping the index in curly braces". How about this:
This shell script will undergo variable substitution via Java's MessageFormat class before it is run on newly started worker machines. MessageFormat will replace special tokens (consisting of numbers inside curly braces) with configuration information specific to the worker being started. These are:
0
1
2